### PR TITLE
Rename the directory `libraries/classes/` to `src/`

### PR DESCRIPTION
**Pull request by MauricioFauth**
- Related to https://github.com/phpmyadmin/phpmyadmin/issues/18512

### CODE REVIEW

File: scripts/check-release-excludes.sh [hunk 1]
```diff
@@ -110,6 +110,11 @@ validateExtension() {
             fi
         ;;
         libraries/*)
```
Comment (williamdes) [inline scripts/check-release-excludes.sh:112]: it should be dropped as this script version is only for master

**Review by williamdes — APPROVED** [review]

File: scripts/check-release-excludes.sh (hunk 1 above)
Comment (MauricioFauth) [reply scripts/check-release-excludes.sh:112]: I left it because there are still some files in it.

Comment (williamdes) [reply scripts/check-release-excludes.sh:112]: Oh okay

**Comment by kamil-tekiela** [comment]
Is there any easy way to rebase my work-in-progress branches to avoid many conflicts?

**Comment by MauricioFauth** [comment]
> Is there any easy way to rebase my work-in-progress branches to avoid many conflicts?

@kamil-tekiela What I usually do is `git checkout my-branch` and `git rebase master`. For me Git has been smart enough to resolve the rename, the only exceptions are the Psalm and PHPStan baselines, that I think it's easier to just regenerate them.
What issues do you have?

**Comment by kamil-tekiela** [comment]
Ok, that worked, but I used cherry-pick instead. Rebasing was too messy for some reason. I picked the commits into a new branch and git handled the dir rename.

**Comment by MauricioFauth** [comment]
Yes. Cherry-picking works great when rebasing is too messy. You want to keep the same branch, you can also create a new branch, go back to the branch, hard reset it, and then cherry-pick the commits.

**Pull request merged**